### ISSUE TITLE
Spells that land after Bubble, Cyclone and Iceblock are immune.

### ISSUE THREAD

**Issue by kysqt** [dalaranwow/dalaran-wow#5246]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Spells that land right after cyclone, bubble and iceblock are immune, this really cripples boomkin burst and cc, as soon as those spells end the target should be able to take damage or cc.

**Expected behaviour**: Spells cast while the target is in bubble, cyclone or Iceblock should still be able to land if timed correctly with the ending of said spell. This is not the case on dalaran-wow.

**Steps to reproduce the problem**:

1. Cyclone, cast damaging spell on target to land right after Cylone duration is over.
2. Divine Shield, cast damaging spell on target to land right after Divine Shield duration is over.
3. Iceblock, cast damaging spell on target to land right after Iceblock duration is over.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by alexuktc** [comment]
This is not an easy issue. It's about the whole system and it affects all classes.
